### PR TITLE
Accounting localization: update mexico 13.0

### PR DESCRIPTION
**Pull request by jcs-odoo**

### CODE REVIEW

File: accounting/fiscal_localizations/localizations/mexico.rst [hunk 1]
```diff
@@ -283,263 +283,346 @@ process in Odoo, this considerations to understand the behavior are important.
...
+   will have the most common labels by default. If the tag you need is not created, you can create
+   it.
 
+Trial Balance
+-------------
 
-Electronic Trial Balance
-------------------------
+Exactly like the COA but with the credit and debit of the initial balance, once you have
+correctly configured your COA, you can go to `Reports --> Trial Balance` this is automatically
```
Comment (Feyensv) [inline accounting/fiscal_localizations/localizations/mexico.rst:384]:
```suggestion
correctly configured your COA, you can go to :menuselection:`Reports --> Trial Balance` this is automatically
```

**Review by Feyensv — COMMENTED** [review]
Little technical remark (shown as `[UNKNOWN NODE title_reference]`) ;).

**Comment by vbe-odoo** [comment]
Hi @jcs-odoo !
LGTM! I can see all the new changes!
Thanks for the hard work!
Bests!
--

File: accounting/fiscal_localizations/localizations/mexico.rst (hunk 1 above)
Comment (jcs-odoo) [reply accounting/fiscal_localizations/localizations/mexico.rst:384]: thanks :)
done

**Review by Feyensv — APPROVED** [review]

**Pull request merged**